src/services/dashboards/widgets/_components/WidgetFrame.vue

```diff
@@ -34,32 +34,52 @@
                 </slot>
             </div>
         </div>
+        <div v-if="editMode"
+             class="edit-mode-cover"
+        >
+            <div class="button-group">
+                <template v-for="icon in editModeIconButtonList">
+                    <p-icon-button v-if="icon.isAvailable"
+                                   :key="icon.name"
```

Comment:
Since the same key will be repeated for each widget, it would be good to put some special value. (like `title`?)

---

위젯들마다 해당 키가 반복될 것이므로, 특수한 값을 넣는 게 좋을듯합니다.